mozilla-mobile/focus-android/app/src/main/res/values/strings.xml

```diff
@@ -135,6 +135,11 @@
 <p>%1$s is produced by Mozilla. Our mission is to foster a healthy, open Internet.<br/>
 <a href="%2$s">Learn more</a></p>]]></string>
 
+    <!-- Toast shown to the user when they are activating the secret dev menu
+    The first parameter is number of long clicks left to enable the menu -->
+    <string name="about_debug_menu_toast_progress">Debug menu: %1$d click(s) left to enable</string>
```

Comment:
@Delphine We have the same functionality and string in Fenix. What do we use there?

Reply:
I see than string in English actually fell through the cracks for Fenix - so languages such as [Russian](https://pontoon.mozilla.org/ru/firefox-for-android/mozilla-mobile/fenix/app/src/main/res/values/strings.xml/?search=Debug+menu&string=209022) had to work around it, the way I suggested.
But this can lead some locales to confusion, not knowing how to work around it.
I'll file a bug for the string in Fenix. But I think in the meantime we should get it fixed in Focus

Reply:
The strings for the Secret Settings, are not intended to be translated. These are enabled on Debug Mode only. See https://github.com/mozilla-mobile/focus-android/issues/6341 for the change that the team has to implement.